src/Node.java

```diff
@@ -7,18 +7,19 @@
 
     public static double sigma = 0;
 
-    private Type nodeType;
     public List<Double> inputs;
     public List<Double> weights;
-    public double output;
 
+    public double output;
     public double mu = 0;
 
+    private Type nodeType;
+
     public Node(Type nodeType) {
         this.nodeType = nodeType;
     }
 
-    public void calculateOutput() {
+    public double calculateOutput() {
```

Comment:
I hate to switch this back but it makes https://github.com/Penchant/RosenbrockFunctionApproximator/pull/23/files#diff-d216b68c5af4435c023be07e5a5352c9R32 so much cleaner